boot/modeenv.go

```diff
@@ -44,6 +44,10 @@ type Modeenv struct {
 
 	// read is set to true when a modenv was read successfully
 	read bool
+
+	// origindir is set to the location where the modeenv was read from, and
+	// where it will be written back to
+	origindir string
```

Comment:
A suggestion I left here https://github.com/snapcore/snapd/pull/8337#discussion_r397649352 was to rename it to `originRootDir`. Then the comment needs an update to indicate that it's the origin root directory, rather than a specific dir where modeenv is located in.